test/extended/builds/start.go

```diff
@@ -409,6 +411,43 @@ var _ = g.Describe("[sig-builds][Feature:Builds][Slow] starting a build using CL
 			})
 
 			g.Describe("start a build via a webhook", func() {
+
+				// AUTH-509: Webhooks do not allow unauthenticated requests by default.
+				// Create a role binding which allows unauthenticated webhooks.
+				g.BeforeEach(func() {
```

Comment:
Note - for clusters upgrading from 4.15 or earlier, this is effectively a no-op. `system:unauthenticated` is granted the `system:webhooks` role globally.